server/server_test.go

```diff
@@ -7751,6 +7756,7 @@ stream
 		.message('message')
 		.details('details')
 		.crit(lambda: TRUE)
+		.sensu()
```

Comment:
The sensu TestServer_AlertHandlers test was failing without that being set. 
`        server_test.go:7784: sensu: unexpected sensu request:
                exp
                [{Name:id Source:Kapacitor Output:message Status:2}]
                got
                []
`
 Is there something I can do to make it work without that?